src/main/util.ts

```diff
@@ -451,6 +451,16 @@ const cutVideo = async (initialFile: string, finalDir: string, desiredDuration:
     });
 }
 
+/**
+ * Gets string value from the config in a more reliable manner.
+ * @param cfg the config store
+ * @param preference the preference
+ * @returns the string config
+ */
+const getStringConfigSafe = (cfg: any, preference: string) : string => {
+  return cfg.has(preference) ? cfg.get(preference) + "/" : "";
```

Comment:
I hated writing the + "/" stuff. Even in early days it felt like a lazy choice :P 
Can we use path.join instead?